src/librssguard/gui/mediaplayer/libmpv/libmpvbackend.cpp

```diff
@@ -54,6 +54,11 @@ LibMpvBackend::LibMpvBackend(Application* app, QWidget* parent)
   mpv_set_option_string(m_mpvHandle, "config", "yes");
   mpv_set_option_string(m_mpvHandle, "script-opts", "osc-idlescreen=no");
   mpv_set_option_string(m_mpvHandle, "hwdec", "auto");
+
+#if defined(MEDIAPLAYER_LIBMPV_OPENGL)

```

Comment:
Tweaked.

Reply:
Thanks! Can you give me some time just to test this on a build with `-DMEDIAPLAYER_LIBMPV_OPENGL=OFF`?

Reply:
Alright, I built and tested with `-DMEDIAPLAYER_LIBMPV_OPENGL=OFF` on Linux (Flatpak) and Wine (again lol), and like you said, specifying `vo=libmpv` isn't necessary in that case.

The OSD still mentions `libmpv` as the display anyway (which I don't quite understand why, but whatever):

![image](https://github.com/martinrotter/rssguard/assets/626206/a732ac00-5085-44d9-91ba-254615c7a409)

Anyway, I've squashed your commit into mine and added you as a co-author, just to make the list of commits "cleaner". Hope it's okay. :)

Reply:
Yes, libmpv sometimes behaves in a very fishy way. Qt-backend in this regard is quite better (althought does not have as many features). OK.